odoo/addons/base/views/ir_qweb_widget_templates.xml

```diff
@@ -33,6 +33,9 @@
                 <span>
                     <div>
                         <span itemprop="addressLocality" t-esc="city"/>,
+                        <t t-if="object.state_id and object.country_id.state_required">
+                            <span itemprop="addressState" t-esc="object.state_id.code"/>,
+                        </t>
                         <span itemprop="addressCountry" t-esc="country_id"/>
```

Comment:
After a bit of testing I think we should do this instead to avoid having `,` when we don't have a country for example

```suggestion
                        <span itemprop="addressLocality" t-esc="city"/>
                        <t t-if="object.state_id and object.country_id.state_required">
                            ,<span itemprop="addressState" t-esc="object.state_id.code"/>
                        </t>
                        <t t-if="country_id">,<span itemprop="addressCountry" t-esc="country_id"/></t>
```